test/pihole.toml

```
@@ -209,6 +209,10 @@
   # Port used by the DNS server
   port = 53
 
+  # Enable/Disable the localise-queries option of dnsmasq. When this setting is disabled 
```

```suggestion
  # Enable/Disable the localise-queries option of dnsmasq. When this setting is disabled
```

I think there is an additional space....